spec/toggles/configuration_spec.rb

```diff
@@ -1,3 +1,5 @@
+require 'spec_helper'
```

Comment:
Yeah, I added them when I was trying to fix some test failures (which actually turned out to be because the `describe Feature::Foo` declaration was trying to load the constant `Feature::Foo` *before* the `before(:each)` block ran, which I fixed by changing all of the `describe`s from constants to strings. I'll remove them.